package.json

```diff
@@ -6,6 +6,13 @@
   "author": "Alexander Rollison",
   "license": "MIT",
   "scripts": {
-    "start": "echo 'Starting extension...' && exit 0"
+    "start": "echo 'Starting extension...' && exit 0",
+    "build": "tsc",
+    "dev": "tsc --watch",
+    "check-types": "tsc --noEmit"
```

Comment:
Run `npm run dev` locally to have the typescript compiler continuously run on new changes